ext/mixed/js/japanese.js

```diff
@@ -80,6 +80,8 @@ function jpConvertReading(expressionFragment, readingFragment, readingMode) {
                 }
             }
             return readingFragment;
+        case 'none':
+            return null;
```

Comment:
Just to confirm: `null` is already handled correctly by anything that uses this functionality?

Reply:
It ends up in Handlebars.js after going through `QueryParser.processParseResultForDisplay`. I didn't see any errors when testing this.